TPL/force-app/main/default/classes/HCCostAccountController.cls

```diff
@@ -162,6 +162,11 @@ public with sharing class HCCostAccountController {
     @AuraEnabled
     public static List<Healthcare_cost__c> assignAll(String currentAccountId,String newCaseId,List<Healthcare_cost__c> currentRecords){
         List<Healthcare_Cost__c> hccList = [select id,case2__c from Healthcare_Cost__c where Account__c =: currentAccountId and id NOT IN: currentRecords and Cost_Include__c = false and Cost_Review__c = false FOR UPDATE];
+        System.debug(newCaseId);
```

Comment:
As an empty string is passed in the parameter, this replaces it with null to avoid an assignment error on the id field.